services/plugins/plugins/schemas.go

```diff
@@ -65,22 +65,47 @@ var pluginSchema = map[string]*schema.Schema{
 	"created_at": {
 		Type:        schema.TypeString,
 		Computed:    true,
-		Description: "When it was created",
+		Description: "When the plugin was created",
+	},
+	"function_name": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Function name with the following format : plugins-UUID.",
 	},
 	"created_by": {
 		Type:        schema.TypeString,
 		Computed:    true,
-		Description: "Who created it",
+		Description: "Who created the plugin",
 	},
-	"last_modified_at": {
+	"source_code_file_id": {
 		Type:        schema.TypeString,
 		Computed:    true,
-		Description: "When it was last modified",
+		Description: "Uploaded file identifier with UUID format.",
+	},
+	"sdk_version": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "SDK version in the format 1.X.X or 2.X.X where X is a number.",
```

Comment:
it would be nice to have a validation, also this is required for command plugins, maybe we could validate that as well